### ISSUE TITLE
Campo "Deceduti"

### ISSUE THREAD

**Issue by joscit** [pcm-dpc/COVID-19#578]
**Tipo di richiesta**:  richiesta di informazione | richiesta di dati | suggerimento di miglioramento (documentazione, label, _ecc_) | relativo alla pagina grafica

Salve a tutti, sto ragionando sul campo "**deceduti**". 

Sarebbe importante conoscere quanti tra i deceduti e tra i dimessi_guariti  derivino da:

* terapia_intensiva
* ricoverati_con_sintomi
* isolamento_domiciliare
* da altri set 

Per esempio oggi <20 aprile 20> ci sono stati 454 nuovi deceduti e 1.822 nuovi guariti. Considerando che le terapie intensive sono diminuite rispetto a ieri di 62 unità, da quale insieme derivano gli altri (454-62) = 392 deceduti ?

**UTILITA'**
Conoscere da dove arrivano i deceduti e incrociare questi dati con le terapie impiegate e con i tempi di inizio terapia permetterebbe di comprendere quanto i deceduti siano dovuti al virus stesso o all'intasamento delle terapie intensive e/o ad una terapia tardiva.

Inoltre un altro dubbio. Il campo **variazione_totale_positivi** dal punto di vista aritmetico è 

variazione_totale_positivi = nuovi_positivi - nuovi_dimessi_guariti - nuovi_deceduti 

Ma dal punto di vista del significato come va letto ? Per esempio quello di oggi è per la prima volta negativo -20. Quindi indica che il numero di nuovi contagiati è inferiore alla somma dei guariti e deceduti..... però il fatto che scenda non significa necessariamente che sia una cosa buona visto che ciò potrebbe essere dovuto ad una prevalenza di nuovi_deceduti..... 

grazie
giuseppe

**Comment by Rabelaiss** [comment]
Concordo, la provenienza dei deceduti e dei guariti sarebbe di indubbia utilità, e sarebbe interessante anche capire la provenienza delle persone in isolamento, dei ricoverati e delle persone in terapia intensiva. Ancora non mi è chiara una cosa:
chi esce dalla terapia intensiva viene sempre mandato a casa o può anche essere "declassato" a ricoverato? e nel primo caso, viene sempre mandato a casa come guarito o può anche essere mandato a casa in isolamento (quindi ancora contagiato ma con sintomi lievi o inesistenti)?

Considerato da solo, il dato `variazione_totale_positivi` non dice niente. Per avere un'utilità deve essere confrontato insieme ai nuovi guariti e decessi.

**Comment by joscit** [comment]
> Concordo, la provenienza dei deceduti e dei guariti sarebbe di indubbia utilità, e sarebbe interessante anche capire la provenienza delle persone in isolamento, dei ricoverati e delle persone in terapia intensiva. Ancora non mi è chiara una cosa:
> chi esce dalla terapia intensiva viene sempre mandato a casa o può anche essere "declassato" a ricoverato? e nel primo caso, viene sempre mandato a casa come guarito o può anche essere mandato a casa in isolamento (quindi ancora contagiato ma con sintomi lievi o inesistenti)?

Ho i tuoi stessi dubbi. Però considera che probabilmente ho commesso un errore. Avevo scritto che 

_Per esempio oggi <20 aprile 20> ci sono stati 454 nuovi deceduti e 1.822 nuovi guariti. Considerando che le terapie intensive sono diminuite rispetto a ieri di 62 unità, da quale insieme derivano gli altri (454-62) = 392 deceduti ?_

Questo è sbagliato perchè la diminuzione delle terapie intensive di 62 unità rispetto al giorno prima rappresentano questo dato (Delta delle terapie intensive)

[ti(oggi)-ti(ieri)]= nuovi_ti - usciti 

quindi se per ipotesi questo incremento è di zero potrebbe essere dovuto al fatto che i nuovi ricoverati in ti siano uguali a quelli usciti.

Ora gli usciti dalla terapia intensiva possono andare nel set degli ospedalizzati o guariti o deceduti. Sarebbe interessante capire come calcolare il dato <usciti> in modo da conoscere il numero dei nuovi ricoverati in terapia intensiva <nuovi_ti>.

Detto questo, in ogni caso, sarebbe interessante conoscere la provenienza dei nuovi deceduti e dei dimessi_guariti . Se provengono da 

- [ ] terapia_intensiva
- [ ] ricoverati_con_sintomi
- [ ] isolamento_domiciliare
- [ ] da altri set

Si potrebbe scoprire per ex. che nel tempo i ricoverati in terapia intensiva diminuiscono a favore dell'isolamento domiciliare e ricoverati. Stessa cosa i dimessi_guariti nel tempo uscirebbero sempre meno dalle terapie intensive..... e magari confrontare questi dati con le terapie impiegate e le tempistiche di inizio delle cure...
